examples/table/sortable-table.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<title>Sortable Table Example | WAI-ARIA Authoring Practices 1.2</title>
+
+<!--  Core js and css shared by all examples; do not modify when using this template. -->
+<link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+<link rel="stylesheet" href="../css/core.css">
+<script src="../js/examples.js"></script>
+<script src="../js/highlight.pack.js"></script>
+<script src="../js/app.js"></script>
+
+<!-- css for this example. -->
+<link href="css/sortable-table.css" rel="stylesheet">
+
+<!--  js for this example -->
+<script src="js/sortable-table.js"></script>
+
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/16">Related Issues</a></li>
+      <li><a href="../../#table">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>Sortable Table Example</h1>
+    <p>
+      The example below illustrates an implementation of the <a href="../../#table">table design pattern</a> for a table with sortable rows.
+      The example uses HTML table markup for all elements of the table structure, e.g., cells, rows, column headers, and caption.
+      The <code>aria-sort</code> attribute is set on the column header of the currently sorted column, and the header text of sortable columns is wrapped in a <code>button</code> element.
+      One column, the <q>Address</q> column is not sortable.
+    </p>
+    <p>Similar examples include: </p>
+    <ul>
+      <li><a href="table.html">Table Example</a>: A table constructed using ARIA roles to convey  table semantics.</li>
+      <li><a href="../grid/dataGrids.html">Data Grid Examples</a>: Three example implementations of grid that include features relevant to presenting tabular information, such as content editing, sort, and column hiding.</li>
+    </ul>
+
+    <section>
+      <h2>Example Option</h2>
+      <label>
+        <input type="checkbox"
+          value="show-unsorted-icon"
+          aria-describedby="show-unsorted-icon-desc">
+        Display sort icon in headers of columns that are not sorted
+      </label>
+      <p id="show-unsorted-icon-desc">
+        Adds an <q>♢</q> icon to the header of each column that is sortable but is not currently sorted.
+        Some implementations do this to help users distinguish sortable columns from columns that cannot be sorted.
+        It is important that a different shaped icon is used for users with visual impairments to perceive the difference with the sorted column icon in high contrast modes.
```

Comment:
What does this mean?  A "different shaped icon". If the icon were the same as the up or down icons, then the icon would be meaningless to all users. I'm not sure whether we should clarify or delete this setence.

Reply:
@mcking65 I made some edits to the example option description for you to review.